.drone.yml

```diff
@@ -59,7 +59,7 @@ steps:
   commands:
   - export CARGO_HOME=$PWD/drone/cargo
   - export CARGO_TARGET_DIR=$PWD/drone/target
-  - export PATH=$CARGO_HOME/bin:$PATH
+  - export PATH=$CARGO_HOME/bin:$CARGO_TARGET_DIR/debug:$PATH
```

Comment:
Is this necessary?

Reply:
yes, because it need to call 'dev-utils', without it, cannot call 'dev-utils'

Reply:
The `run_multinode.sh` will add the target dir into PATH, it'll also take `BUILD_PROFILE` into account.

Reply:
but, when i run drone local, PATH didn't work.
also this PATH is static, i think it's simpler just to include inside .drone.yml

if it's located to other script, hard to catch bug when running error occurs